grails-app/i18n/messages.properties

```diff
@@ -691,6 +691,8 @@ email.contactAdmin.message=Please contact your local OpenBoxes administrator wit
 email.contactAdminReceiving.message=Please contact the receiver listed above or your local OpenBoxes administrator with any questions you have about the status of this item. If you see a quantity in the cancelled column, this means that the warehouse received fewer items than expected, possibly as a result of a data error or a loss during shipment. The receiver can explain the situation. If you have an OpenBoxes account, you can view the shipment via
 email.thisLink.label=this link
 email.requestReceived.message=You have received a new request from: {0}, created by: {1}. Click <a href="{2}">here</a> to review your requests pending your approval.
+email.statusChange.message=Your request status has been updated to: {0}. Click <a href="{1}">here</a> to review your requests.
+email.withComment.message=With comment:
```

Comment:
I know @jmiranda was not a big fan or adding some `:` or commas in the message itself, but I'm fine with that.

Reply:
You have a point, this should probably not be in the message label